invenio_rdm_records/records/mappings/v7/rdmrecords/drafts/draft-v5.0.0.json

```diff
@@ -634,6 +733,72 @@
             "type": "keyword"
           }
         }
+      },
+      "files": {
+        "type": "object",
+        "properties": {
+          "enabled": {
+            "type": "boolean"
+          },
+          "default_preview": {
+            "type": "keyword"
+          },
+          "order": {
+            "type": "keyword"
+          },
+          "entries": {
```

Comment:
if entries are not indexed, removed. so we do it in the future when we reach this task